product_docs/docs/pem/9/upgrading/upgrading_pem_installation/high_availability_upgrade_with_httpd.mdx

```diff
@@ -0,0 +1,47 @@
+---
+title: "Upgrading a PEM installation on high-availability mode using HTTPD"
+---
+
+If you are using [Failover Manager for high availability](considerations/setup_ha_using_efm/) on your PEM server, you can perform an upgrade of the PEM installation by updating the PEM packages.
+
+The example uses these IP addresses:
+
+-  Primary server: 172.17.0.2
+-  Secondary server: 172.17.0.3
+-  Monitored server: 172.17.0.4
```

Comment:
Note to self > clarify what happens with this third server. Upgrade like with secondary?